engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.h

```diff
@@ -150,6 +150,7 @@ FLUTTER_DARWIN_EXPORT
 
 // UITextInputTraits
 @property(nonatomic) UITextAutocapitalizationType autocapitalizationType;
+@property(nonatomic) UITextAutocapitalizationType originalAutocapitalizationType;
```

Comment:
can add a comment explaining why adding this? (why not just use `autocapitalizationType`?)